spec/models/child_benefit_tax_calculator_spec.rb

```diff
@@ -556,7 +556,7 @@
         children_count: 1,
         starting_children: {
           "0" => {
-            start: { day: "24", month: "06", year: "2013" },
+            start: { day: "01", month: "07", year: "2013" },
```

Comment:
The name of the test is "should give an accurate figure for 40 weeks at £20.30"

Under the new Monday rules, 24/06/2013 would have been included, and the benefit amount would be calculated for 41 weeks not 40. 

I chose to change the date rather than the name of the test and the result because I felt that by not setting the end date the spirit test implied that it was testing the last six months of the tax year.

The original commit note states the nature of the change and how the tests have been updated to match them. I didn't think I needed to do a more granular per test commit.